### PR TITLE
Fix the limzers, weh!

### PR DESCRIPTION
**Pull request by DogZeroX**
## About the PR 

**Screenshots**

**Changelog**

:cl:
- tweak: Reptilians no longer have human limbs.
- tweak: Reptilians now need to breathe.
- add: Reptilians now have their own SKREEE!

### CODE REVIEW

File: Resources/Prototypes/Body/Parts/reptilian.yml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+#    deadThreshold: 120
+  - type: Input
+    context: "ghost"
+  - type: DummyInputMover
+  - type: GhostOnMove
+
+- type: entity
+  id: LeftArmReptilian
+  name: "left reptilian arm"
+  parent: PartHuman
```
Comment (Fishfish458) [inline Resources/Prototypes/Body/Parts/reptilian.yml:67]: Shouldn't this be parenting PartReptilian?

File: Resources/Prototypes/Body/Parts/reptilian.yml [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    compatibility: Biological
+    symmetry: Left
+#    criticalThreshold: 40
+#    deadThreshold: 80
+
+- type: entity
+  id: RightArmReptilian
+  name: "right reptilian arm"
+  parent: PartHuman
+  components:
```
Comment (Fishfish458) [inline Resources/Prototypes/Body/Parts/reptilian.yml:88]: Parenting PartReptilian?

File: Resources/Prototypes/Body/Parts/reptilian.yml (hunk 1 above)
Comment (DogZeroX) [reply Resources/Prototypes/Body/Parts/reptilian.yml:67]: checking

File: Resources/Prototypes/Body/Parts/reptilian.yml [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+    compatibility: Biological
+    symmetry: Right
+#    criticalThreshold: 45
+#    deadThreshold: 90
+
+- type: entity
+  id: LeftFootReptilian
+  name: "left reptilian foot"
+  parent: PartHuman
+  components:
```
Comment (Fishfish458) [inline Resources/Prototypes/Body/Parts/reptilian.yml:188]: PartReptilian?

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**